tests/test_caption.py

```diff
@@ -19,29 +17,15 @@
         ],
     ],
 )
-@patch(
-    'inference_client.client.get_model_spec',
-    Mock(return_value={'endpoints': {'grpc': 'grpc://mock.inference.jina.ai'}}),
-)
-@patch('inference_client.client.login', Mock(return_value='valid_token'))
-@patch(
-    'inference_client.base.BaseClient._post',
-    Mock(
-        return_value=DocumentArray(
-            [
-                Document(
-                    uri='https://picsum.photos/id/233/100',
-                    tags={'response': 'a image of two rails'},
-                ),
-            ]
-        )
-    ),
-)
-def test_caption_document(inputs):
-    model = Client().get_model('mock-model')
+def test_caption_document(make_flow, inputs):
+    model = BaseClient(
```

Comment:
can we make this into contest? seems that this model is used for multiple test files